src/ts/ConvivaAnalytics.ts

```diff
@@ -128,6 +134,7 @@ export class ConvivaAnalytics {
 
     // Set default config values
     this.config.debugLoggingEnabled = this.config.debugLoggingEnabled || false;
+    this.config.deviceCategory = this.config.deviceCategory || Conviva.Client.DeviceCategory.WEB;
```

Comment:
I'd probably prefer to set a default DeviceCategory in `Html5Metadata` class. I feel like if ConvivaAnalytics is refactored and someone removes this OR condition, then DeviceCategory would no longer be sent over in the Conviva beacon.

I'd consider seeing if `this.config.deviceCategory` is set, and if so, to set the property on `Html5Metadata` otherwise defer to the class default.

Reply:
Makes sense, I updated that.